runtime/reference/cli/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Deno CLI Subcommands"
+oldUrl: "/runtime/reference/cli/"
+---
+
+The Deno CLI (Command Line Interface) allows you to interact with the Deno
+runtime environment from your terminal or command prompt. The CLI has a number
+of subcommands that can be used to perform different tasks, check the links
+below for more information on each subcommand.
+
+- [deno add](/runtime/reference/cli/add) - Add dependencies
+- [deno bench](/runtime/reference/cli/benchmarker/) - benchmarking tool
+- [deno check](/runtime/reference/cli/check/) - type check your program without
+  running it
+- [deno compile](/runtime/reference/cli/compiler/) - compile a program into a
+  standalone executable
+- [deno completions](/runtime/reference/cli/completions/) - generate shell
+  completions
+- [deno coverage](/runtime/reference/cli/coverage/) - generate test coverage
+  reports
+- [deno doc](/runtime/reference/cli/documentation_generator/) - generate
+  documentation for a module
+- [deno eval](/runtime/reference/cli/eval/) - evaluate provided script
+- [deno fmt](/runtime/reference/cli/formatter/) - format your code
+- [deno info](/runtime/reference/cli/dependency_inspector/) - inspect an ES
+  module and all of its dependencies
+- [deno init](/runtime/reference/cli/init/) - create a new project
+- [deno install](/runtime/reference/cli/install/) - install a dependency or a
+  script
+- [deno jupyter](/runtime/reference/cli/jupyter/) - run a Jupyter notebook
+- [deno lint](/runtime/reference/cli/linter/) - lint your code
+- [deno lsp](/runtime/reference/cli/lsp/) - language server protocol integration
+- [deno publish](/runtime/reference/cli/publish/) - publish a module
+- [deno remove](/runtime/reference/cli/remove) - Remove dependencies
+- [deno repl](/runtime/reference/cli/repl/) - starts a read-eval-print-loop
+- [deno run](/runtime/reference/cli/run/) - run a script
+- [deno serve](/runtime/reference/cli/serve/) - run a web server
+- [deno task](/runtime/reference/cli/task_runner/) - run a task
+- [deno test](/runtime/reference/cli/test/) - run your tests
+- [deno types](/runtime/reference/cli/types/) - print runtime types
+- [deno uninstall](/runtime/reference/cli/uninstall/) - uninstall a dependency
+  or a script
+- [deno upgrade](/runtime/reference/cli/upgrade/) - upgrade Deno to the latest
+  version
+- [environment variables](/runtime/reference/cli/env_variables/)
+- [unstable feature flags](/runtime/reference/cli/unstable_flags/)
```

Comment:
Match the organization in `deno help` as @bartlomieju suggested?

```
~# deno help
Deno: A modern JavaScript and TypeScript runtime

Usage: deno [OPTIONS] [COMMAND]

Commands:
  Execution:
    run          Run a JavaScript or TypeScript program, or a task
                  deno run main.ts  |  deno run --allow-net=google.com main.ts  |  deno main.ts
    serve        Run a server
                  deno serve main.ts
    task         Run a task defined in the configuration file
                  deno task dev
    repl         Start an interactive Read-Eval-Print Loop (REPL) for Deno
    eval         Evaluate a script from the command line

  Dependency management:
    add          Add dependencies
                  deno add @std/assert  |  deno add npm:express
    install      Install script as an executable
    uninstall    Uninstall a script previously installed with deno install
    remove       Remove dependencies from the configuration file

  Tooling:
    bench        Run benchmarks
                  deno bench bench.ts
    cache        Cache the dependencies
    check        Type-check the dependencies
    compile      Compile the script into a self contained executable
                  deno compile main.ts  |  deno compile --target=x86_64-unknown-linux-gnu
    coverage     Print coverage reports
    doc          Genereate and show documentation for a module or built-ins
                  deno doc  |  deno doc --json  |  deno doc --html mod.ts
    fmt          Format source files
                  deno fmt  |  deno fmt main.ts
    info         Show info about cache or info related to source file
    jupyter      Deno kernel for Jupyter notebooks
    lint         Lint source files
    init         Initialize a new project
    test         Run tests
                  deno test  |  deno test test.ts
    publish      Publish the current working directory's package or workspace
    upgrade      Upgrade deno executable to given version
                  deno upgrade  |  deno upgrade 1.45.0  |  deno upgrade canary


Environment variables:
  DENO_AUTH_TOKENS      A semi-colon separated list of bearer tokens and hostnames
                        to use when fetching remote modules from private repositories
                         (e.g. "abcde12345@deno.land;54321edcba@github.com")
  DENO_FUTURE           Set to "1" to enable APIs that will take effect in Deno 2
  DENO_CERT             Load certificate authorities from PEM encoded file
  DENO_DIR              Set the cache directory
  DENO_INSTALL_ROOT     Set deno install's output directory
                         (defaults to $HOME/.deno/bin)
  DENO_NO_PACKAGE_JSON  Disables auto-resolution of package.json
  DENO_NO_UPDATE_CHECK  Set to disable checking if a newer Deno version is available
  DENO_TLS_CA_STORE     Comma-separated list of order dependent certificate stores.
                        Possible values: "system", "mozilla".
                         (defaults to "mozilla")
  HTTP_PROXY            Proxy address for HTTP requests
                         (module downloads, fetch)
  HTTPS_PROXY           Proxy address for HTTPS requests
                         (module downloads, fetch)
  NO_COLOR              Set to disable color
  NO_PROXY              Comma-separated list of hosts which do not use a proxy
                         (module downloads, fetch)
  NPM_CONFIG_REGISTRY   URL to use for the npm registry.

Docs: https://docs.deno.com
Standard Library: https://jsr.io/@std
Bugs: https://github.com/denoland/deno/issues
Discord: https://discord.gg/deno
```

Reply:
I will do that in another PR, just to avoid confusion with the menu changes made in this PR, in case it needs to be reverted.